rosys/analysis/timelapse_recorder.py

```diff
@@ -150,7 +157,7 @@ def _save_image(image: RosysImage, path: Path, size: tuple[int, int], notificati
     for message in notifications:
         y += 30
         _write(message, draw, x, y)
-    img.save(path / f'{image.time:.3f}.jpg', 'JPEG')
+    img.save(path / f'{image.time:.3f}_{image.camera_id[-5:].upper()}.jpg', 'JPEG')
```

Comment:
I don't think that we need to reduce the camera ID to the "human readable" 2-byte snippets for internal filenames. But it's not a show-stopper for me.

Reply:
True. I updated the code accordingly.